cumulusci/tasks/bulkdata/snowfakery.py

```diff
@@ -503,6 +507,7 @@ def data_generator_opts(self, working_dir, *args, **kwargs):
             "continuation_file": wd.continuation_file,
             "num_records_tablename": self.run_until.sobject_name or COUNT_REPS,
             "vars": self.recipe_options,
+            "plugin_options": plugin_options,
```

Comment:
Just wanted to double check that it's intentional this is configurable in the old task but hardcoded in the new one.

Reply:
It's configurable in the old task as an implementation detail of making it available to the new one. Making it configurable here is a reasonable feature request and I can do that before the next CCI release but probably shouldn't scope creep today.